templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -168,6 +176,18 @@ Parameters:
     ConstraintDescription: 'Must be a valid Bitbucket version number. For example: 5.0.0 or higher'
     Description: 'Version of Bitbucket to install. Please use version 5.0.0 or higher. Find valid versions at http://go.atlassian.com/bbs-releases'
     Type: String
+  BitbucketLicenseKey:
+    Default: ''
+    Description: (Optional) Provide a license key for Bitbucket Data Center if you have one. 
+    Type: String
+  BitbucketAdminPassword:
+    Description: (Optional) Password for the administrator ('admin') account. Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, 1 (non / @ " ') symbol. Optional - If none provided, the template will use the value for the DBPassword.
```

Comment:
There was a recent change to the supported characters description:


```
    Description: "Password for the master ('postgres') account. Must be at least 8 characters and include 1 uppercase, 1 lowercase, 1 number, and 1 of the following symbols: ! # $ { * : [ = , ] - _ @ + % &"
```
https://github.com/atlassian/quickstart-atlassian-bitbucket/blob/develop/templates/quickstart-bitbucket-dc.template.yaml#L382

I am not sure if it is a good strategy to reuse DB password for admin as well, I think people always should provide the password if they want to setup admin. Otherwise they can miss the description and they will be confused how to log in. I also don't think it is a good security practice to recycle the password.

Reply:
agreed, will make password non optional